Jun_Youngsang/css/style.css

```diff
@@ -0,0 +1,215 @@
+/*
+Welcome to the style.css file
+########################################################################
+Contents:
+body
+#map
+#header
+#features
+#footer
+styling for the header
+styling for the features
+styling for the footer
+styling for any popups
+styling for various classes used in the storymap
+*/
+
+/* The body element is the parent element of all other elements. */
+body {
+    margin: 0;
+    padding: 0;
+}
+
+/* The map element is the container for the map. */
+#map {
+    top: 0;
+    width: 100%;
+    height: 100vh;
+    position: fixed;
+}
+
+/* Styling any popups */
+.mapboxgl-popup-content p {
+    font-weight: 300;
+    margin-top: 0.3em;
+    margin-bottom: 0;
+}
+
+.mapboxgl-popup-content h4 {
+    font-weight: 500;
+    font-size: 0.9em;
+    border-width: 0 0 0.5px;
+    border-style: solid;
+    border-color: rgb(80 80 80);
+    margin-top: 0.5em;
+    margin-bottom: 0.5em;
+}
+
+/* The header will hold the title, byline and an introductory description of the project. */
+#header {
+    margin: auto;
+    min-height: 100vh; 
+    width: 100%; 
+    position: relative; 
+    z-index: 5; /* Ensures the header is always on top of the map */
+    padding: 2vh 0 5vh; /* top right bottom left */
+  }
+
+/* The features element holds all the chapters. */
+#features {
+    padding-top: 80vh;
+    padding-bottom: 10vh;
+  }
+
+@media (width <= 750px) { 
+    #features { 
+        width: auto;
+        margin: 10vh;
+    }
+  } /* This would be for media with a maximum width of 750px */
```

Comment:
**Suggestion:** Currently some components of your story map have mobile compatibility and some haven't. Ideally, you should make this consistent. For this assignment, mobile compatibility is not required. Besides, you can also combine the two `@media` components into one at the end.